src/extras/primitives/primitives/a-text.js

```diff
@@ -0,0 +1,28 @@
+/* Experimental text primitive.
+ */
+var getMeshMixin = require('../getMeshMixin');
+var registerPrimitive = require('../primitives').registerPrimitive;
+var utils = require('../../../utils/');
+
+registerPrimitive('a-text', utils.extendDeep({}, getMeshMixin(), {
+  defaultComponents: {
+    'text': {anchor: 'align', width: 5}
+  },
+  mappings: {
+    align: 'text.align',
```

Comment:
These could be automatically populated given the schema. I'd also add a hyphen whenever there's a camelCase (e.g., `font-image`).